VMEncryption/main/check_util.py

```diff
@@ -168,10 +149,41 @@ def validate_volume_type(self, public_settings):
         if not volume_type.lower() in map(lambda x: x.lower(), supported_types) :
             raise Exception("Unknown Volume Type: {0}, has to be one of {1}".format(volume_type, supported_types))
 
+    def validate_lvm_os(self, public_settings):
+        volume_type = public_settings.get(CommonVariables.VolumeTypeKey).lower()
```

Comment:
The "get" from public settings may return null.
And we may want to skip the check for operation that's not an enable-type operation (not one of Enable, EnableEncryptFOrmat, EFA)

Reply:
added logic and tests to handle this